config/services-prod.yml

```diff
@@ -375,6 +375,62 @@ https://cmr.earthdata.nasa.gov:
           exists: [ 'variableSubset', 'dimensionSubset' ]
       - image: !Env ${GEOLOCO_IMAGE}
 
+  - name: asdc/imagenator_l2
+    description: Chain to create TEMPO L2 imagery for GIBS
+    data_operation_version: '0.22.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/harmony/imagenator_l2
+    umm_s: SD3611093087-LARC_CLOUD
+    capabilities:
+      concatenation: false
+      subsetting:
+        bbox: false
+        variable: true
+        temporal: false
+      output_formats:
+        - image/png
+      reprojection: false
```

Comment:
Does using the swath projector mean that it can reproject to different projections? Or is the swath projector just being used to always convert to the same projection, and we wouldn't let a user request specify a different projection?